yal/surveys/facebook_invite.py

```diff
@@ -0,0 +1,117 @@
+from vaccine.base_application import BaseApplication
+from vaccine.states import Choice, EndState, WhatsAppButtonState
+from yal import config
+from yal.utils import get_generic_error
+
+
+class Application(BaseApplication):
+    START_STATE = "state_facebook_member"
+    NOT_INTERESTED_STATE = "state_facebook_invite_decline"
+
+    async def state_facebook_member(self):
+        async def _next(choice: Choice):
+            if choice.value == "no":
+                return "state_not_a_member"
+            return "state_was_a_member"
+
+        return WhatsAppButtonState(
+            self,
+            question=self._(
+                "Great! Before we get started, have you been a member of the B-Wise "
+                "Facebook page since before June 2023?"
+            ),
+            choices=[
+                Choice("yes", "Yes"),
+                Choice("no", "No"),
+            ],
+            next=_next,
+            error=self._(get_generic_error()),
+        )
+
+    async def state_was_a_member(self):
+        return EndState(
+            self,
+            self._(
+                "\n".join(
+                    [
+                        "Excellent! Thank you for being a valuable member of our "
+                        "Facebook community.",
+                        "",
+                        "To do the survey, click on the link below, answer the "
+                        "questions and you’re all done!",
+                        "",
+                        config.FACEBOOK_SURVEY_INVITE_MEMBERS_URL,
+                    ]
+                )
+            ),
+            next=self.START_STATE,
```

Comment:
next state shouldn't be start state

Reply:
I think thats the standard for `EndState`